### PR TITLE
Add a note for DIR parameters

### PR DESCRIPTION
**Pull request by core-man**
References:
- https://docs.generic-mapping-tools.org/6.1/cookbook/features.html#directory-parameters
- https://docs.generic-mapping-tools.org/6.2.0rc2/gmt.conf.html#dir-parameters

**Preview**: https://gmt-china.github.io/sitepreview/gmt-china/GMT_docs/dir-parameter/conf/dir/

### CODE REVIEW

File: source/conf/dir.rst [hunk 1]
```diff
@@ -1,22 +1,30 @@
 DIR参数
 =======
 
+.. note::
```
Comment (core-man) [inline source/conf/dir.rst:4]: This is translated from https://docs.generic-mapping-tools.org/6.1/cookbook/features.html#directory-parameters:

> GMT versions prior to GMT 5 relied solely on several environment variables ($GMT_SHAREDIR, $GMT_DATADIR, $GMT_USERDIR, and $GMT_TMPDIR), pointing to folders with data files and program settings. Beginning with version 5, some of these locations are now (also or exclusively) configurable with the gmtset utility. When an environment variable has an equivalent parameter in the gmt.conf file, then the parameter setting will take precedence over the environment variable.

File: source/conf/dir.rst [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
+
+   GMT 5 之前的 GMT 版本仅依赖于几个环境变量（\ **$GMT_SHAREDIR**\ 、\ **$GMT_DATADIR**\ 、\ 
+   **$GMT_USERDIR** 和 **$GMT_TMPDIR**\ ），用于指向数据文件和程序设置的目录。
+
+   自 GMT 5 开始，一些目录\ **只**\ 或者\ **同时**\ 使用 :doc:`gmtset` 进行配置。 
+   若这些环境变量在 GMT 配置文件 :file:`gmt.conf` 中有等效参数，那么配置文件中的参数设置将优先于环境变量。
+
 下面列出所有与目录相关的参数，参数的默认值在中括号内列出。
 
 .. glossary::
```
Comment (core-man) [inline source/conf/dir.rst:14]: Some update from https://docs.generic-mapping-tools.org/6.1/gmt.conf.html#dir-parameters

File: source/conf/dir.rst [hunk 3]
```diff
@@ -1,22 +1,30 @@
 DIR参数
 =======
 
+.. note::
+
+   GMT 5 之前的 GMT 版本仅依赖于几个环境变量（\ **$GMT_SHAREDIR**\ 、\ **$GMT_DATADIR**\ 、\ 
+   **$GMT_USERDIR** 和 **$GMT_TMPDIR**\ ），用于指向数据文件和程序设置的目录。
+
+   自 GMT 5 开始，一些目录\ **只**\ 或者\ **同时**\ 使用 :doc:`gmtset` 进行配置。
```
Comment (core-man) [inline source/conf/dir.rst:9]:
```suggestion
   自 GMT 5 开始，一些目录\ **只**\ 或者\ **同时**\ 使用 :doc:`/module/gmtset` 进行配置。 
```

File: source/conf/dir.rst [hunk 4]
```diff
@@ -1,22 +1,30 @@
...
     **DIR_DCW**
-        DCW数据文件的路径，默认值为空，GMT会自动猜测合理的路径值。
+        :doc:`DCW数据 </dataset/dcw/index>` 所在路径，默认值为空。GMT 会自动猜测合理的路径值。
 
     **DIR_GSHHG**
-        海岸线数据所在路径
-
-    **DIR_CACHE**
-        用于存储从GMT服务器下载的临时文件（以 **@** 开头）的目录 [~/.gmt/cache]
+        :doc:`GSHHG 数据 </dataset/gshhg>` 所在路径。若该参数为空，则路径默认为 :file:`**$GMT_SHAREDIR**\ /coast` 目录。
```
Comment (core-man) [inline source/conf/dir.rst:30]:
```suggestion
        :doc:`GSHHG 数据 </dataset/gshhg>` 所在路径。若该参数为空，则路径默认为 **$GMT_SHAREDIR**\ /coast 目录。
```

File: source/conf/dir.rst [hunk 5]
```diff
@@ -1,22 +1,30 @@
...
     **DIR_DATA**
-        存放数据文件的目录，默认值为空。
+        数据文件的存放目录，默认值为空。覆盖环境变量 **$GMT_DATADIR** 的值。
 
-        GMT在命令中遇到文件名时，首先在当前目录下寻找该文件，
-        若找不到，则会到参数 **DIR_DATA** 指定的目录中寻找，
-        若找不到，则到环境变量 **$GMT_DATADIR** 所指定的目录中寻找。
+        GMT 在命令中遇到文件名时，首先在当前目录下寻找该文件。
+        若找不到，则会到参数 **DIR_DATA** 指定的目录中寻找。
+        若找不到，则到环境变量 **$GMT_DATADIR** 指定的目录中寻找。
```
Comment (wangliang1989) [inline source/conf/dir.rst:24]:
```suggestion
        若依然找不到，则到环境变量 **$GMT_DATADIR** 指定的目录中寻找。
```

File: source/conf/dir.rst [hunk 6]
```diff
@@ -1,22 +1,30 @@
 DIR参数
 =======
 
+.. note::
+
+   GMT 5 之前的 GMT 版本仅依赖于几个环境变量（\ **$GMT_SHAREDIR**\ 、\ **$GMT_DATADIR**\ 、\ 
+   **$GMT_USERDIR** 和 **$GMT_TMPDIR**\ ），用于指向数据文件和程序设置的目录。
+
+   自 GMT 5 开始，一些目录\ **只**\ 或者\ **同时**\ 使用 :doc:`gmtset` 进行配置。 
+   若这些环境变量在 GMT 配置文件 :file:`gmt.conf` 中有等效参数，那么配置文件中的参数设置将优先于环境变量。
```
Comment (wangliang1989) [inline source/conf/dir.rst:10]: 这一行和前一行的内容似乎缺乏逻辑联系

**Review by wangliang1989 — COMMENTED** [review]
我对DIR参数不了解

File: source/conf/dir.rst (hunk 6 above)
Comment (core-man) [reply source/conf/dir.rst:10]: 有一定联系，说明配置参数优先级高于环境变量。

File: source/conf/dir.rst [hunk 7]
```diff
@@ -1,22 +1,30 @@
 DIR参数
 =======
 
+.. note::
+
+   GMT 5 之前的 GMT 版本仅依赖于几个环境变量（\ **$GMT_SHAREDIR**\ 、\ **$GMT_DATADIR**\ 、\ 
+   **$GMT_USERDIR** 和 **$GMT_TMPDIR**\ ），用于指向数据文件和程序设置的目录。
+
+   自 GMT 5 开始，一些目录\ **只**\ 或者\ **同时**\ 使用 :doc:`/module/gmtset` 进行配置。 
+   若这些环境变量在 GMT 配置文件 :file:`gmt.conf` 中有等效参数，那么配置文件中的参数设置将优先于环境变量。
```
Comment (seisman) [inline source/conf/dir.rst:10]: 我们应尽量避免在文档中提到 GMT5。

Comment (core-man) [reply source/conf/dir.rst]: removed in https://github.com/gmt-china/GMT_docs/pull/350/commits/e178dbe02c67191d3518fbf9ada8622e6eadccc7

File: source/conf/dir.rst [hunk 8]
```diff
@@ -1,22 +1,30 @@
 DIR参数
 =======
 
+.. note::
+
+   早期，GMT 仅依赖于几个环境变量（\ **$GMT_SHAREDIR**\ 、\ **$GMT_DATADIR**\ 、\ 
+   **$GMT_USERDIR** 和 **$GMT_TMPDIR**\ ），用于指向数据文件和程序设置的目录。
+
```
Comment (seisman) [inline source/conf/dir.rst:8]: 我的意思是，不介绍任何 GMT 的旧行为，而是只介绍 GMT 的当前行为。没有历史负担。

Comment (core-man) [reply source/conf/dir.rst]: See the updating in 30229cb.

File: source/conf/dir.rst [hunk 9]
```diff
@@ -1,22 +1,26 @@
 DIR参数
 =======
-
+   
+可以使用 :doc:`/module/gmtset` 设定指向数据文件和程序设置的目录。
```
Comment (seisman) [inline source/conf/dir.rst:4]: > 可以使用 :doc:`/module/gmtset` 设定指向数据文件和程序设置的目录。

This sentence reads weird.

Comment (core-man) [inline source/conf/dir.rst:4]:
```suggestion
可以使用 :doc:`/module/gmtset` 设定一些专用的目录位置，如数据文件和程序设置的目录。
这时，我们只需指定文件名，GMT 会自动去这些目录中找同名文件。
```

Comment (core-man) [reply source/conf/dir.rst:4]: See the revision below.

**Review by seisman — APPROVED** [review]

**Pull request merged**